### DISCUSSION TITLE
Java集合框架

### DISCUSSION THREAD

**Discussion by giscus** [mxz94/mxz_back#383] [General]
# Java集合框架

Astrofy is a free and open-source template for your Personal Portfolio Website built with Astro and TailwindCSS. Create in minutes a website with Blog, CV, Project Section, Store and RSS Feed.

https://mxz-back.pages.dev/note/Java%E9%9B%86%E5%90%88%E6%A1%86%E6%9E%B6/

**Comment by mxz94** [comment]
测试评论一

**Comment by bubble1994** [comment]
1111
